lib/nanoc/base/views/item_rep_view.rb

```diff
@@ -27,6 +27,11 @@ def name
       @item_rep.name
     end
 
+    # @return [Boolean]
```

Comment:
In other words, could you add `# @api private` so it’s marked as private in the API docs?